uniswap_universal_router_decoder/_encoder.py

```diff
@@ -872,6 +883,42 @@ def permit2_transfer_from(
         self._add_command(RouterFunction.PERMIT2_TRANSFER_FROM, args)
         return self
 
+    def permit2_transfer_from_batch(
+            self,
+            transfer_details: List[Permit2TransferDetail]) -> _ChainedFunctionBuilder:
```

Comment:
Please change `transfer_details` into `batch_details`, here and in the docstring, to stay consistent with Uniswap's contracts